locales/es.json

```diff
@@ -1,72 +1,72 @@
 {
-	"select-user-type.who_you_are": "¿Cual es su perfil? (Selecciona por ahora una opción, esto lo podrá cambiar posteriormente)",
+	"select-user-type.who_you_are": "¿Cuál es su perfil? (Seleccione por ahora una opción. Esta la podrá cambiar posteriormente)",
 	"select-user-type.driver": "Conductor",
 	"select-user-type.passenger": "Pasajero",
-	"request-phone.type_your_phone": "Por favor envían tu número telefónico. No vamos a compartir tu información.",
+	"request-phone.type_your_phone": "Por favor envíe su número de teléfono. No compartiremos su información.",
 	"request-phone.button_text": "Enviar número",
-	"request-phone.all_set": "👌 Estamos listos, ¡Ya esta listo para ordenar un viaje ahora!",
-	"passenger-index.choose_taxi_type": "¿Que tipo de taxi desea?",
+	"request-phone.all_set": "👌 Todo preparado. ¡Ya esta listo para ordenar un viaje ahora!",
```

Comment:
Yes .I'll do it . My number is 707-712-3175 thank you for doing this and helping me earning money .